flink-container/src/main/java/org/apache/flink/container/entrypoint/StandaloneJobClusterEntryPoint.java

```diff
@@ -67,6 +68,8 @@
 	@Nonnull
 	private final SavepointRestoreSettings savepointRestoreSettings;
 
+	private final JobID jobID = new JobID(Long.MIN_VALUE, Long.MIN_VALUE);
```

Comment:
The resulting `JobID` for this is: `80000000000000008000000000000000`.

What do you think about using `new JobID(0, 0)` to give an all zero `JobID` instead? Since this is a fixed ID anyways, I think going with the simplest ID is preferable and an all zero ID might be easier to use when manually working with the REST API of a job instead of remembering to insert an `8` (which also looks very similar to a `0`).